### DISCUSSION TITLE
如何将节点直接放置到边上

### DISCUSSION THREAD

**Discussion by laobieyi** [antvis/X6#2644] [Q&A]
下面是我想实现的效果，在我拖拽到画布上的时候是正常，即如下：
![image](https://user-images.githubusercontent.com/110290751/188819312-5ec138be-37a0-4901-a4e4-0cbac17ff0e0.png)
但是在我反序列化加载数据到画布上时就变成如下图：
![image](https://user-images.githubusercontent.com/110290751/188819796-715c3f71-105a-46a3-8092-a8d31510b4a7.png)
请问怎么解决？

**Comment by zmcode** [comment]
用默认的路由https://x6.antv.vision/zh/examples/edge/router#normal

**Reply by laobieyi** [reply]
可以啦，谢谢
